Release Cognitive Services Data Plane SDKs Round I (#4955)
Azure/azure-sdk-for-js: sdk/cognitiveservices/cognitiveservices-imagesearch/README.md

#### nodejs - Authentication, client creation and search images as an example written in TypeScript.

##### Install @azure/ms-rest-azure-js

```bash
npm install @azure/ms-rest-azure-js
```

##### Sample code

> The following sample lets you perform an image search for 'Microsoft Azure' with conditions such as the color has to be 'Monochrome', etc. To know more, Refer the [Azure Documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/bing-image-search/)

Reviewer: ```suggestion
The following sample performs an image search for 'Microsoft Azure' with conditions such as the color has to be 'Monochrome', etc. To know more, refer to the [Azure Documentation on Bing Image Search](https://docs.microsoft.com/en-us/azure/cognitive-services/bing-image-search/)
```
